### PR TITLE
feat(feedback onboarding): add angular, astro, ember docs

### PR DESCRIPTION
**Pull request by michellewzhang**
Adds Angular, Astro, and Ember onboarding docs for user feedback
Relates to https://github.com/getsentry/sentry/issues/62792

### CODE REVIEW

File: static/app/gettingStartedDocs/javascript/angular.tsx [hunk 1]
```diff
@@ -293,9 +302,47 @@ const replayOnboarding: OnboardingConfig<PlatformOptions> = {
   nextSteps: () => [],
 };
 
+const feedbackOnboarding: OnboardingConfig<PlatformOptions> = {
+  install: (params: Params) => [
+    {
+      type: StepType.INSTALL,
+      description: tct(
+        'For the User Feedback integration to work, you must have the Sentry browser SDK package, or an equivalent framework SDK (e.g. [code:@sentry/react]) installed, minimum version 7.85.0.',
```
Comment (ryan953) [inline static/app/gettingStartedDocs/javascript/angular.tsx:310]:
```suggestion
        'For the User Feedback integration to work, you must have the Sentry browser SDK package, or an equivalent framework SDK (e.g. [code:@sentry/angular]) installed, minimum version 7.85.0.',
```

Comment (ryan953) [reply static/app/gettingStartedDocs/javascript/angular.tsx:310]: could also be `[codeAngular:@sentry/angular] or [codeIvy:@sentry/angular-ivy]` as the replay snippet above

File: static/app/gettingStartedDocs/javascript/astro.tsx [hunk 2]
```diff
@@ -228,8 +230,49 @@ const replayOnboarding: OnboardingConfig = {
   nextSteps: () => [],
 };
 
+const feedbackOnboarding: OnboardingConfig = {
+  install: () => [
+    {
+      type: StepType.INSTALL,
+      description: tct(
+        'For the User Feedback integration to work, you must have the Sentry browser SDK package, or an equivalent framework SDK (e.g. [code:@sentry/react]) installed, minimum version 7.85.0.',
```
Comment (ryan953) [inline static/app/gettingStartedDocs/javascript/astro.tsx:238]: ditto above

```suggestion
        'For the User Feedback integration to work, you must have the Sentry browser SDK package, or an equivalent framework SDK (e.g. [code:@sentry/astro]) installed, minimum version 7.85.0.',
```

File: static/app/gettingStartedDocs/javascript/ember.tsx [hunk 3]
```diff
@@ -184,8 +193,46 @@ const replayOnboarding: OnboardingConfig = {
   nextSteps: () => [],
 };
 
+const feedbackOnboarding: OnboardingConfig = {
+  install: () => [
+    {
+      type: StepType.INSTALL,
+      description: tct(
+        'For the User Feedback integration to work, you must have the Sentry browser SDK package, or an equivalent framework SDK (e.g. [code:@sentry/react]) installed, minimum version 7.85.0.',
```
Comment (ryan953) [inline static/app/gettingStartedDocs/javascript/ember.tsx:201]:
```suggestion
        'For the User Feedback integration to work, you must have the Sentry browser SDK package, or an equivalent framework SDK (e.g. [code:@sentry/ember]) installed, minimum version 7.85.0.',
```

File: static/app/gettingStartedDocs/javascript/angular.tsx (hunk 1 above)
Comment (ryan953) [reply static/app/gettingStartedDocs/javascript/angular.tsx:310]: or we could rephrase this sentence... i don't have a strong suggestion for that though, just an option. I was looking at how it's phrased for the react onboarding and it's a bit different, which is fine to be the same or different too.

**Review by ryan953 — APPROVED** [review]

File: static/app/gettingStartedDocs/javascript/angular.tsx (hunk 1 above)
Comment (michellewzhang) [reply static/app/gettingStartedDocs/javascript/angular.tsx:310]: i'm copying what's in the docs, but will incorporate the first suggestion for sure

**Pull request merged**